lib/board.rb

```diff
@@ -1,11 +1,106 @@
 class Board
 
+  attr_reader :size, :grid
+
+  FIRST_ELEMENT = 0
+  NUMBER_ONE = 1
+  NUMBER_TWO = 2
+
   def initialize(size)
     @size = size
     @grid = create_grid
   end
 
+  public
+
   def create_grid
-    (0..@size - 1).to_a
+    (NUMBER_ONE..@size).to_a
+  end
+
+  def mark_board(number, mark)
+    @grid[number - NUMBER_ONE] = mark
+  end
+
+  def possible_rows
```

Comment:
Ruby has a great method called `transpose` in the Array class.

Check it out, it will simplify making rows into columns.